hoodie-cli/pom.xml

```diff
@@ -167,11 +167,30 @@
       <artifactId>log4j</artifactId>
       <version>${log4j.version}</version>
     </dependency>
+
+    <dependency>
+      <!-- TODO: Need to refactor to move SchemaUtil.readSchemaFromLogFile from hoodie-hive -->
+      <groupId>com.uber.hoodie</groupId>
+      <artifactId>hoodie-hive</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+
     <dependency>
       <groupId>com.uber.hoodie</groupId>
       <artifactId>hoodie-client</artifactId>
       <version>${project.version}</version>
     </dependency>
+
```

Comment:
the common theme with these changes is to explicitly specify direct dependencies.